bread/views/userprofile.py

```diff
@@ -292,7 +292,7 @@ def get_object(self):
 class EditPermissionsView(EditView):
     model = get_user_model()
     fields = ["is_active", "is_superuser", "is_staff"]
-    urlparams = {}
+    urlparams = ()
```

Comment:
I think we should add ```urlparams: Iterable[Tuple[str, Type]] = ()``` in ```bread.views.util.BreadView``` instead. This feature is used just so the URL paths can be generated automatically for class-based views.